### ISSUE TITLE
kendo-drawing - upgrade to 1.9.1 causes errors in PROD build (optimization: true)

### ISSUE THREAD

**Issue by kunajs** [telerik/kendo-angular#3091]
**Describe the bug**
In my code I do nothing special, just similar code which is used in [Basic Shapes](https://www.telerik.com/kendo-angular-ui/components/drawing/basic-shapes/) examples. In fact, I just want to draw a line, so I use Path class:
```
@ViewChild('surface', { static: false }) surfaceElement: ElementRef<HTMLDivElement>;
// in ngAfterViewInit
const surface = Surface.create(this.surfaceElement.nativeElement);
const path = new Path(stroke: { color: "#9999b6", width: 3 });
path.moveTo(0, 0).lineTo(150, 0).lineTo(150, 65).lineTo(0, 65).close();
surface.draw(path);
```

With version 1.8.1 (and later I found out that up to version 1.9.0) everything works fine.
Then I **upgraded to 1.9.1**. When served on localhost, without some extra build flags, everything works fine too. The problem occures only when build is run with option **"optimization": "true"** (in angular.json) - which we use for production build.

The `draw` method fails with error on this line in `svg/node.js` file: 
`const childNode = new NODE_MAP[srcElement.nodeType](srcElement, this.options);`
When debugged, srcElement.nodeType is "Path", but **NODE_MAP object is empty**!!! The error message is:
`E[o.nodeType] is not a constructor`

I have suspicion that it has to do something with the fact that `"sideEffects": "true"` was changed to `"false"` in your `package.json` file.

**To Reproduce**
Create app with kendo-drawing 1.9.1 and draw rectangle on surface.
Serve it on localhost as usual - no problem.
Serve it on localhost with "optimization": "true" - problem!

**Expected behavior**
Should work with "optimization": "true".

**Browser**
 - OS: Windows 10
 - Chrome
 - 86.0.4240.75

**Comment by tsvetomir** [comment]
Thanks for reporting this problem. It wasn't apparent when testing with components that use kendo-drawing like the Charts.

Should be fixed in the latest [development builds](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by tsvetomir** [comment]
Released in v1.9.3; @kunajs let me know if you still experience the issue with this version

**Comment by kunajs** [comment]
Seems to be ok. Thanks.
